config/settings/production.yml

```diff
@@ -3,13 +3,13 @@ publish_api_url: https://api.publish-teacher-training-courses.service.gov.uk
 publish_url: https://www.publish-teacher-training-courses.service.gov.uk
 find_url: https://www.find-postgraduate-teacher-training.service.gov.uk
 
-base_url: https://api.publish-teacher-training-courses.service.gov.uk
+base_url: https://www2.publish-teacher-training-courses.service.gov.uk
```

Comment:
We may need to updated DSI config (via their admin panel) to make this a valid redirect URL. 

Relevant setup in this repo: https://github.com/DFE-Digital/teacher-training-api/blob/master/config/initializers/omniauth.rb#L17

Reply:
Dan has already done this now

Reply:
yes and sandbox